read_input.l

```diff
@@ -61,9 +61,10 @@ EQL {SPC}*={SPC}*
 #include"meas/measurements.h"
 #include"integrator.h"
 #include"operator.h"
+#include"quda_interface.h"
 #include"phmc.h"
 #include<io/params.h>
-
+#define QUDA_MAX_MG_LEVEL 5
```

Comment:
This should certainly be in another file, such as ```quda_defines.h```, such that it is easy to find.